.github/workflows/ci.yml

```diff
@@ -126,11 +126,24 @@ jobs:
             v1-go-${{ runner.os }}
       - name: Set up Go
         uses: actions/setup-go@v3
+      - name: Build go
+        working-directory: go-grpc-gateway
+        run: ./install_tools.sh && ./build.sh
```

Comment:
I think you need to install protoc judging from the error in gha?

Reply:
I couldn't find a Docker image with `golang + protoc + cargo` (all of which this job requires). It might be best to add golang to our builder image: https://github.com/mobilecoinofficial/docker-rust-sgx-base

Reply:
Alternatively, we could try https://github.com/arduino/setup-protoc